tests/o11y/o11y.go

```diff
@@ -0,0 +1,65 @@
+package o11y
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+)
+
+const (
+	O11yUser = "o11y-e2e"
+)
+
+var _ = framework.O11ySuiteDescribe("OllY E2E tests", Label("o11y", "HACBS"), Pending, func() {
+
+	defer GinkgoRecover()
+	var f *framework.Framework
+	var err error
+
+	Describe("O11y test", Pending, func() {
+		var testNamespace string
+
+		BeforeAll(func() {
+
+			f, err = framework.NewFramework(O11yUser)
+			Expect(err).NotTo(HaveOccurred())
+			testNamespace = f.UserNamespace
+
+			_, err := f.AsKubeAdmin.CommonController.CreateTestNamespace(testNamespace)
+			Expect(err).NotTo(HaveOccurred(), "Error when creating/updating '%s' namespace: %v", testNamespace, err)
+
+		})
+		AfterAll(func() {
+			if !CurrentSpecReport().Failed() {
+				// err := f.AsKubeAdmin.CommonController.DeleteAllPipelineRunsAndTasks(testNamespace)
+				// Expect(err).NotTo(HaveOccurred())
+				Expect(f.SandboxController.DeleteUserSignup(f.UserName)).NotTo(BeFalse())
+			}
+		})
+
+		It("E2E sample test for upload 50MB pod", Pending, func() {
+			podNameRegex := ".*-upload-50mb-pod"
+			query := fmt.Sprintf("last_over_time(container_network_transmit_bytes_total{namespace='%s', pod=~'%s'}[1h])", testNamespace, podNameRegex)
```

Comment:
Shouldn't we use the query we have defined in the rule file?
`appstudio_container_network_transmit_bytes_total`

Reply:
OK, Thanks Sri